### PR TITLE
[DO NOT MERGE] Add hybrid link swagger and examples.

### PR DESCRIPTION
**Pull request by nemijato**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet. - During February.
  
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet. - When feature goes to public preveiw - Estimate September.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [x] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by nemijato** [comment]
I need ARM team to sign off on it, so I can proceed with the arm manifest update.

**Comment by ericshape** [comment]
@nemijato 
your side would re-publish this API again?
1. now, we switch to our API commit to a dev branch: https://portal.azure-devex-tools.com/branch/Azure/azure-rest-api-specs/dev-sql-Microsoft.Sql-2020-11-01-preview...master/   please follow https://github.com/Azure/azure-rest-api-specs/pull/12146 to set your upstream to `dev-sql-Microsoft.Sql-2020-11-01-preview`

2. can you send an email or internal chat to me about your release schedule? it will determine when "dev-sql-Microsoft.Sql-2020-11-01-preview" branch merge to the master branch of API Swagger repo.

3. if your endpoint does not have a strict target version request (like we commit to the customer that we should release it in 2020-02-02), can you put it in the 2020-11-01 version in the swagger file?

**Review by ericshape — CHANGES_REQUESTED** [review]
please set the upstream to dev-sql-Microsoft.Sql-2020-11-01-preview

**Review by ericshape — APPROVED** [review]
approve it because the author needs ARM team to sign off for ARM manifest deployment.

Please don't merge it to master. it should be merged to dev branch

**Review by bcham — APPROVED** [review]

**Comment by ericshape** [comment]
@nemijato 
have you completed the ARM approval process with this dummy PR? Once you have done that, can you close this PR? Thanks!

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/HybridLinks.json
Comment (chiragg4u) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/HybridLinks.json:291]: >Resource [](start = 5, length = 8)

Please refer to common definitions "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/..."

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/HybridLinks.json
Comment (chiragg4u) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/HybridLinks.json:166]: >delete [](start = 7, length = 6)

Please add PATCH atleast for the tags field.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/examples/HybridLinkListByInstance.json
Comment (chiragg4u) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/examples/HybridLinkListByInstance.json:26]: this would have nextlink right?

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/HybridLinks.json
Comment (chiragg4u) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/HybridLinks.json:318]: >"$ref": "#/definitions/Resource" [](start = 10, length = 32)

You are removing the required field from the resource and that's why pointing to common field will help making it correct,

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/HybridLinks.json
Comment (nemijato) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/HybridLinks.json:166]: I don't need patch approach in my feature.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/examples/HybridLinkListByInstance.json
Comment (nemijato) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/examples/HybridLinkListByInstance.json:26]: Yeah, I'll add more items in this example.

**Pull request closed without merging**
